Tools/ardupilotwaf/ardupilotwaf.py

```diff
@@ -60,6 +60,7 @@
     'AP_ICEngine',
     'AP_Frsky_Telem',
     'AP_FlashStorage',
+    'AP_UAVCAN',
```

Comment:
Well, I counted 16 of libraries that I do not use in my plane...
Anyway, I agree that waf should be tuned and made correct.